package.json

```diff
@@ -50,13 +50,13 @@
     "jest": "24.1.0",
     "metro-react-native-babel-preset": "0.54.1",
     "react": "16.8.6",
-    "react-native": "0.60.0-rc.3",
+    "react-native": "0.60.4",
     "react-test-renderer": "16.8.6"
   },
   "resolutions": {
     "handlebars": "4.1.2",
     "js-yaml": "3.13.1",
-    "@react-native-community/cli": "^2.0.0-rc.5"
+    "lodash": "4.17.13"
```

Comment:
All of these resolutions are likely not necessary. Make sure they're latest possible in the lock file and then hopefully remove

Reply:
I don't really want Github to scream at me each time I visit this repo :<

Reply:
Remove yarn.lock and add again, should make them gone :D but if not, keep the resolutions field

Reply:
Not sure until we merge, but yeah, I'll try that ^^